### PR TITLE
Pfem fluid make script object

### PR DESCRIPTION
**Pull request by maceligueta**
The file KratosPfemFluid.py file is recovered. Contains the same operations as the main script that is being used now, but it is now an object called Solution (a class). This object can be called from outside, by a coupling algorithm, and it can be derived to modify its behaviour according to the needs of the coupling algorithm.

### CODE REVIEW

**Comment by josep-m-carbonell** [comment]
@maceligueta, I could not test it today, tomorrow I will try it. It seems ok. I pre approve it because it do not have conflicts with anything and it looks ok. Thank you MA.

**Review by josep-m-carbonell — APPROVED** [review]

File: applications/PfemFluidDynamicsApplication/python_scripts/KratosPfemFluid.py [hunk 1]
```diff
@@ -1,234 +1,355 @@
...
+        solver_module = __import__(self.ProjectParameters["solver_settings"]["solver_type"].GetString())
+        self.solver = solver_module.CreateSolver(self.main_model_part, self.ProjectParameters["solver_settings"])
+
+
+        #### Output settings start ####
+
+        self.problem_path = os.getcwd()
+        self.problem_name = self.ProjectParameters["problem_data"]["problem_name"].GetString()
+        
+        self.SetGraphicalOutput()
```
Comment (josep-m-carbonell) [inline applications/PfemFluidDynamicsApplication/python_scripts/KratosPfemFluid.py]: It raises an errror, self.computing_model_part is needed and it is created in the Initilialize...the position of this call must be after it.

**Review by josep-m-carbonell — CHANGES_REQUESTED** [review]

File: applications/PfemFluidDynamicsApplication/python_scripts/KratosPfemFluid.py (hunk 1 above)
Comment (maceligueta) [reply applications/PfemFluidDynamicsApplication/python_scripts/KratosPfemFluid.py]: Yes, I had it corrected in another branch. I just committed the fix. Thanks.

**Review by josep-m-carbonell — APPROVED** [review]

**Comment by josep-m-carbonell** [comment]
Thsnk you MA 👍

**Pull request merged**